### ISSUE TITLE
"Polling" option

### ISSUE THREAD

**Issue by stephaneguindon** [tgvaughan/icylog#5]
Hi Tim. I'm still a big fan of IcyLog but the "polling" option does no longer seem to work on my side. It is probably due to a broken linux install on my side but I can't find a way to make this work again. Would you mind checking whether it could be an IcyLog problem? Cheers.

**Comment by tgvaughan** [comment]
Hi @stephaneguindon, sorry for the slow reply on this.  The polling option definitely won't work any more under Firefox: they have explicitly forbidden the behaviour it relies on.  I'll try to look at this now.

**Comment by tgvaughan** [comment]
Hi again Stephane, it looks like Chrome no longer allows the behaviour of FileReader required to implement the polling.  Firefox disabled this a while ago, as I said.  I've just tested the Reload function in icytree under Chrome and found that it no longer works there either.  Not really surprising, I guess, as I learned some time after implementing this feature that the behaviour in Chrome (and old Firefox) is now considered out of spec.  They explicitly don't allow javascript code to reload files because of security concerns.

Sorry I can't be more helpful!
